src/storage/store/dictionary_column.cpp

```diff
@@ -183,13 +183,21 @@ bool DictionaryColumn::canOffsetCommitInPlace(const ChunkState& offsetState,
     if (offsetState.metadata.compMeta.canAlwaysUpdateInPlace()) {
         return true;
     }
+    CompressionMetadata::InPlaceUpdateLocalState localUpdateState{};
```

Comment:
check to make sure these are declared in the right place, add `[[maybe_unused]]` as needed